subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseWtpIntegrationTest.groovy

```diff
@@ -37,10 +37,10 @@ dependencies {
         libsDir.mkdir()
         libsDir.createFile("foo.jar")
 
-        // when
+        when:
```

Comment:
❌ Even though it's written in Groovy, this still seems like a JUnit test that does not have the `when:` and `then:` constructs.